README.md

```diff
@@ -1,13 +1,28 @@
-# clojure-tictactoe
+# TIC TAC TOE
+----------
+ - Play on a 3x3 or 4x4 board
+ - Choose your marker
+ - Multiple player options
+   - Play against a friend
+   - try to beat the computer (bet you can't!)
 
-Tic-tac-toe! The traditional strategy game where you try to get three markers in a row before your opponent.
 
-## Installation
+## Getting Started:
 
-Built in Clojure. To install Clojure, follow the instructions from (https://leiningen.org/). 
+ 1. If you don’t already have Clojure installed, follow the instructions from (https://leiningen.org/)
```

Comment:
I think you can install clojure via homebrew now. Would be nice to just have the homebrew command here for installing quickly.